### ISSUE TITLE
Add yaml support for 10.10

### ISSUE THREAD

**Issue by kameghamegha** [kameghamegha/macos_configuration#14]

**Comment by sylvainsf** [comment]
Could you elaborate? Is this an issue for finding the 10.10 options for each existing config feature?

**Comment by kameghamegha** [comment]
Yes, sorry. I was just quickly writing these up as notes, but yes, it is find the 10.10 options for each existing config feature (if any).
